docs/getting-started.md

```diff
@@ -40,6 +40,8 @@ should work with Antrea, starting with version 7.4.
 In case a node does not have a supported OVS module installed,
 you can install it following the instructions at:
 [Installing Open vSwitch](https://docs.openvswitch.org/en/latest/intro/install).
+Please be aware that `vport-stt` module is not shipped in built-in kernel module,
+please load it manually if STT tunnel is enabled.
```

Comment:
I think you mean that the kernel module is not in the Linux tree and needs to be built from source. IMO, it's different from a built-in kernel module which is a module which is automatically part of the kernel. For example the openvswitch kernel module is in-tree but is not built-in.

Reply:
yes, I will update with more precise description.